index.js

```diff
@@ -1173,6 +1173,12 @@ class GithubScm extends Scm {
                 ref: hoek.reach(webhookPayload, 'ref')
             };
         case 'release': {
+            const action = hoek.reach(webhookPayload, 'action');
+
+            if (!['published'].includes(action)) {
```

Comment:
can we maybe save the permitted release event hook as a constant var?

Reply:
I saved the permitted release event hook as a constant var.